client/components/UpperCarousel.jsx

```diff
@@ -1,6 +1,32 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
+import styled from 'styled-components';
 import Button from './Button';
 
+const StyledUpperCarousel = styled.div`
+  display: grid;
+  grid-template-columns: 48px auto 48px;
+  grid-template-rows: 100%;
```

Comment:
You are using the same grid template twice. Maybe use inheritance? 
`const StyledUpperCarousel = styled(GridContainer)`

Reply:
Good catch! I like it. I'll implement it on the next branch.

Reply:
Hey, good catch. I like all of your suggestions. I've already started the next feature branch, so I'll implement them there.